### ISSUE TITLE
Hspec version not constrained

### ISSUE THREAD

**Issue by MaxGabriel** [yesodweb/yesod-scaffold#58]
I created scaffolded project with the SQLite backend today using yesod-bin 1.4.3.3, then installed into my global package database. The test suite fails with this compile error:

```
test/TestImport.hs:21:12:
    Not in scope: type constructor or class ‘SpecWith’
cabal: Error: some packages failed to install:
ForeignKeys-0.0.0 failed during the building phase. The exception was:
ExitFailure 1
```

I have HSpec 1.11.4 installed in my global package, which doesn't export `SpecWith` (it looks to have been [introduced in HSpec 2](https://hackage.haskell.org/package/hspec-2.0.0)). Afaict, hspec isn't required by another package and its version [isn't constrained in the cabal file](https://github.com/yesodweb/yesod-scaffold/blob/6dd9d71ba59eb9222448829c284bc88ebf211a16/PROJECTNAME.cabal#L127). Should the version of hspec that comes with the scaffolding be constrained to >= 2.0.0?

**Comment by snoyberg** [comment]
Yes, would you mind sending a PR for this?

**Comment by MaxGabriel** [comment]
Fixed by #59
